### PR TITLE
IBMASC - read 8-bit ASCII tape file.

### PR DESCRIPTION
**Pull request by larsbrinkhoff**
The flight is delayed.

### CODE REVIEW

**Comment by eswenson1** [comment]
Meaning you aren’t ready to merge?

File: doc/programs.md [hunk 1]
```diff
@@ -113,6 +113,7 @@
 - HOSTS3, the host table compiler.
 - HSNAME, displays user's HSNAME.
 - HSNDEV, HSNAME device.
+- IBMASC, read 8-bit ASCII tape file.
```
Comment (atsampson) [inline doc/programs.md]: Read an IBM tape into an ASCII file? (CTSS interchange?)

Comment (larsbrinkhoff) [reply doc/programs.md]: From the name, I suppose it was meant to work with tapes from an IBM machine, but of course it works equally well with any machine that stores 8-bit characters that way.

I don't know whether CTSS used 8-bit ASCII.

**Comment by larsbrinkhoff** [comment]
> Meaning you aren’t ready to merge?

The "todo" label?  No, that's the label I use to mean "this is something to add because ITS used to have it".  As opposed to a new feature, or bug fix, or a program imported from elsewhere.

**Comment by eswenson1** [comment]
No. Meaning “the flight is delayed”.

**Comment by larsbrinkhoff** [comment]
Oh that!  It means I was sitting at an airport with some extra time on my hands.

**Pull request merged**